recipes/mne-qt-browser/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "mne-qt-browser" %}
+{% set version = "0.1.1" %}
```

I updated the package once more.
```suggestion
{% set version = "0.1.2" %}
```